### PR TITLE
Added _supportedLayout attribute

### LINKED ISSUE #28: _supportedLayout attribute needs adding to schema
This is to help the authoring tool determine layout styles for this component

### PR DESCRIPTION
**Pull request by darylhedley**
Includes a version bump too
fixes #28

### CODE REVIEW

File: properties.schema [hunk 1]
```diff
@@ -4,6 +4,11 @@
   "id": "http://jsonschema.net",
   "$ref": "http://localhost/plugins/content/component/model.schema",
   "properties":{
+    "_supportedLayout": {
+      "type": "enum",
+      "required": true,
+      "default": ["full-width", "half-width"]
```
Comment (dennis-learningpool) [inline properties.schema]: should also have the editorOnly option?

Comment (dennis-learningpool) [inline properties.schema]: think this should be:
````
"_supportedLayout": {
      "enum": ["full-width", "half-width"],
      "required": true,
      "default": "full-width",
      "editorOnly": true
}
````
or not?

Comment (darylhedley) [inline properties.schema]: Oops - just added that in

**Comment by brian-learningpool** [comment]
+1

**Comment by dancgray** [comment]
+1

**Comment by dennis-learningpool** [comment]
+1

**Comment by brian-learningpool** [comment]
+1

**Pull request merged**